examples/knative/go/kn-go-tagging/README.md

```diff
@@ -0,0 +1,117 @@
+# kn-go-tagging
+
+Example Knative Go function for attaching or detaching a tag on a vCenter VM that sends a power on/off [CloudEvent](https://github.com/cloudevents/sdk-go) via the VMware Event Broker.
+
+The [ko](https://github.com/google/ko) tool will be used to create the artifacts.
+
+> **Note:** This guide assumes that you have stood up a working Knative environment using the vCenter Event Broker Appliance (VEBA).
+
+# Step 1 - Build
+
+> **Note:** This step is only required if you made code changes to any of the \*.go files.
+
+`ko` was created to help Knative developers build images and binaries without the need for Dockerfiles nor Makefiles.
+
+See more information in [this Knative blog](https://knative.dev/blog/articles/ko-fast-kubernetes-microservice-development-in-go/).
+
+Follow setup instructions at https://github.com/google/ko.
+
+You will also need to install [Go](https://golang.org/doc/install) for the ko tool to work.
+
+After installing ko, set the destination for images with an environment variable.
+
+```bash
+KO_DOCKER_REPO=my-dockerhub-user
+```
+
+Run the following command from the `kn-go-tagging` directory. It will build and push the image to your local Docker daemon. To have `ko` publish and use a container image registry, remove the `--local` flag.
+
+```bash
+ko publish --local .
+```
+
+Save the image name so you can use it in the [function.yaml]'s Service container image section (mentioned in Step 3).
+
+# Step 2 - Tests
+
+// TODO: put command here to run tests
```

Comment:
TODO?

Reply:
Good catch. I'll put the command, probably something like `go test` in the readme.